documentation/modules/overview/con-configuration-points-bridge.adoc

```diff
@@ -11,6 +11,18 @@ Kafka Bridge consumer and producer configuration is standard, as described in th
 
 HTTP-related configuration options set the port connection which the server listens on.
 
+[discrete]
+== CORS
+
```

Comment:
It's maybe worth stating that CORS is optional.

Reply:
Good idea. I've added _If you choose to use CORS, ..._